packages/system76-power-deb/system76-power-deb.pacscript

```diff
@@ -0,0 +1,10 @@
+name="system76-power-deb"
+gives="system76-power"
+breaks="${gives}"
+version="1.1.23"
```

Comment:
Outdated version, `1.1.24` is the latest.

Reply:
Not available in the ppa

Reply:
Get it from here I guess: https://apt-origin.pop-os.org/release/pool/jammy/system76-power/cee2d561326176396832fa4c1d3edf61556d2dba/